tests/test_backend.py

```diff
@@ -3323,7 +3323,7 @@ def func(x, z):
 
     @check_opset_min_version(11, "SpaceToBatchND")
     def test_space_to_batchnd_non_const_7d(self):
-        x_type, y_type, z_type = np.int64, np.int64, np.int64
+        x_type, y_type, z_type = np.float32, np.int64, np.int64
```

Comment:
The black_shapes include `[2, 2]` in this case, so it appears the code is properly tested.